modin/experimental/core/execution/native/implementations/omnisci_on_native/test/test_dataframe.py

```diff
@@ -2140,6 +2140,18 @@ def drop_rename_concat(df1, df2, lib, **kwargs):
             force_arrow_execute=True,
         )
 
+    def test_drop_row(self):
+        def drop_row(df1, df2, lib, **kwargs):
```

Comment:
No need for unused args. You can use `def drop_row(df, **kwargs)` and skip `data2` for `run_and_compare`.

Reply:
Fixed.